Source/Blazorise.Bootstrap/Styles/components/_bar.scss

```diff
@@ -1,9 +1,9 @@
-﻿.navbar-nav .nav-item:hover *:not(:has(.disabled)):not(.disabled) {
+﻿.navbar-nav .nav-link:hover {
```

Comment:
With this change it makes the buttons to have the correct cursor, but that's because non of the elements in BarItem has the cursor pointer (except buttons, because they have them "naturally").

```html
<Bar>
  <BarStart>
    <BarItem>              
      <Button Color="Color.Primary" Disabled>test 1, not ok previously, ok no</Button>
      <Button Color="Color.Primary">test 2, ok</Button>               
    </BarItem>

      <BarItem>              
      <Div> 
          I had cursor-pointer, but has normal cursor now.
      </Div>
    </BarItem>
  </BarStart>
  <BarItem>
    <Button Color="Color.Primary" Disabled>test 3 , ok</Button>     
  </BarItem>
</Bar>
```
So we either:
- discard the functionality of having the BarItem to always has cursor pointer (on divs, buttons, everything) => disabled buttons will always work, but it's quite a breaking change. (this is the solution with your last SCSS structure commit)
- Place the cursor pointer only when there is no disabled button =>  buttons work as expected unless wild structure is introduced. Anything else works as before. Bit of breaking change too, because the `cursor:pointer;` style is not directly on BarItem (.nav-item), but on the descendants..

Reply:
`BarItem` should NOT have the pointer cursor. The `BarLink` should have it. So why not? Because, the BarItem is a container for links, buttons, dropdowns, and whatever else the user can place in it. If it had the cursor pointer, it would be wrong from a UX perspective as it would indicate the wrong action to the user.

Reply:
But previously (before this pr) the BarItem had the cursor pointer. 

removing this
```css
.navbar-nav .nav-item:hover {
    cursor: pointer;
}
```

will:
- solve the OP's issue
- make it right from ux perspective (as you described)
- introduce a breaking change

Reply:
> * introduce a breaking change

Yea, we should do it. I might have accidentally introduced that bug in 2020. As far as I can see from Git history.

Reply:
Ok. I agree. 
As a breaking change -> should I edit the release note, or what's the procedure?

I will also check the link, as it seems there is some unnecessary css.

Reply:
> As a breaking change -> should I edit the release note, or what's the procedure?

Yes. But close this PR and make it for 1.7 instead.